distributed/protocol/tests/test_protocol.py

```diff
@@ -137,6 +137,23 @@ def test_large_messages():
     assert (msg['y']['a'].data == msg2['y']['a']).all()
 
 
+@slow
```

Comment:
This test seems to run quickly enough for me.

```python
In [1]: from distributed.protocol.tests.test_protocol import dumps, loads                                          

In [2]: %time x = {i: 'mystring_%d' % i for i in range(100000)}                                                    
CPU times: user 37.7 ms, sys: 3.42 ms, total: 41.2 ms
Wall time: 40.6 ms

In [3]: %time b = dumps(x)                                                                                         
CPU times: user 40.9 ms, sys: 99 µs, total: 41 ms
Wall time: 40.7 ms

In [4]: x2 = loads(b)                                                                                              

In [5]: %time x2 = loads(b)                                                                                        
CPU times: user 35.4 ms, sys: 324 µs, total: 35.7 ms
Wall time: 34.7 ms

In [6]: %time assert x == x2                                                                                       
CPU times: user 4.94 ms, sys: 97 µs, total: 5.04 ms
Wall time: 5.08 ms
```

We can probably drop the `@slow` decorator